File: EzDeal/API .NET/EzDeal.Domain/Usuario.cs
using System.Collections.Generic;

namespace EzDeal.Domain
{
    public class Usuario
    {
        public long Id {get; set;}
        public char Rol {get; set;}
        public string Email {get; set;}
        public string Contraseña {get; set;}
        public string Nombres {get; set;}
        public string ApellidoPaterno {get; set;}
        public string ApellidoMaterno {get; set;}
        public string Distrito {get; set;}
        public string Direccion {get; set;}
        public string Provincia {get; set;}
        public string Departamento {get; set;}
        public int Strikes {get; set;}
        public string CuentaHabilitada {get; set;}
        public string imagenPerfil {get; set;}
        public Cuenta Cuenta {get; set;}
        public Anunciante InfoAnunciante {get; set;}

        public ICollection<Servicio> Servicios {get; set;}
        public ICollection<Solicitud> Solicitudes {get; set;}




    }
}

File: EzDeal/ezdeal-frontend/src/models/usuario.js
export default class Usuario {
    constructor(
        nombreUsuario, email, contrasena,
        nombres,apellidoPaterno, apellidoMaterno,
        departamento, distrito, direccion,
        provincia) {
      this.nombreUsuario = nombreUsuario;
      this.email = email;
      this.contrasena = contrasena;
      this.nombres = nombres;
      this.apellidoPaterno = apellidoPaterno;
      this.apellidoMaterno = apellidoMaterno;
      this.departamento = departamento;
      this.distrito = distrito;
      this.direccion = direccion;
      this.provincia = provincia;
    }
  }